dev/deps/spark-deps-hadoop-3-hive-2.3

```diff
@@ -205,9 +205,13 @@ objenesis/3.2//objenesis-3.2.jar
 okhttp/3.12.12//okhttp-3.12.12.jar
 okio/1.14.0//okio-1.14.0.jar
 opencsv/2.3//opencsv-2.3.jar
+opentracing-api/0.33.0//opentracing-api-0.33.0.jar
+opentracing-noop/0.33.0//opentracing-noop-0.33.0.jar
+opentracing-util/0.33.0//opentracing-util-0.33.0.jar
 orc-core/1.7.3//orc-core-1.7.3.jar
 orc-mapreduce/1.7.3//orc-mapreduce-1.7.3.jar
 orc-shims/1.7.3//orc-shims-1.7.3.jar
+org.jacoco.agent/0.8.5/runtime/org.jacoco.agent-0.8.5-runtime.jar
```

Comment:
jacoco is Java code coverage library, I was surprised that it would become a dependency

Reply:
oh, I missed this. Do you want to exclude this as an workaround, @sunchao ?

Reply:
Hmm let me check.

Reply:
This is a test-only dependency brought in by `aliyun-java-sdk-core` in `hadoop-cloud-storage`.

Reply:
Are you sure it's test only? Didn't think those appeared in these drops file

Reply:
I tracked to this commit which introduced it: https://github.com/aliyun/aliyun-openapi-java-sdk/commit/e0d21a34ca4e5d3caa82bd1ebdf8e60e8b4cd5ee, which looks only used in test?